inference-engine/src/vpu/graph_transformer/src/stages/broadcast.cpp

```diff
@@ -111,11 +116,13 @@ void FrontEnd::parseBroadcast(
     const auto output = outputs[0];
 
     const auto modeString = layer->GetParamAsString("mode", "numpy");
-    if (modeString == "numpy") {
+    if (modeString == "numpy" || modeString == "bidirectional") {
+        std::cout << "here\n";
         VPU_THROW_UNLESS(inputs.size() == 2,
-                         "{} layer with name {} and numpy mode must have 2 inputs, actually "
-                         "provided {} inputs", layer->type, layer->name, inputs.size());
+                         "{} layer with name {} and {} mode must have 2 inputs, actually "
+                         "provided {} inputs", layer->type, layer->name, modeString, inputs.size());
     } else if (modeString == "explicit") {
```

Comment:
I think it's also better to throw there if the mode is not `numpy`, `bidirectional` or `explicit`.

Reply:
There is final `else` at 133 line that does it